pkg/cloudprovider/providers/gce/gce_loadbalancer_internal.go

```diff
@@ -79,19 +79,29 @@ func (gce *GCECloud) ensureInternalLoadBalancer(clusterName, clusterID string, s
 		return nil, err
 	}
 
+	// Determine IP which will be used for this LB. If no forwarding rule has been established
+	// or specified in the Service spec, then requestedIP = "".
+	requestedIP := determineRequestedIP(svc, existingFwdRule)
```

Comment:
Before reaching this point, I assume it's not possible to have an existing forwarding rule with the *wrong* type of address, e.g., using an EXTERNAL IP. I am relying on the similar logic in external LB.

In the future, if we ever combine external and internal LB code, we'd need to ensure we still check whether the forwarding rule has all the expected attributes (and delete it if not).

Reply:
We could check to see if the address is RFC 1918? (totally breaks down for IPv6...)